interface/src/lib.rs

```diff
@@ -34,36 +35,56 @@ mod wasm;
 
 use solana_pubkey::Pubkey;
 
-// inline some constants to avoid dependencies
+// Inline some constants to avoid dependencies.
```

Comment:
As of today there's a `solana-sdk-ids` crate in the Agave repo so you can use `solana_sdk_ids::sysvar::recent_blockhashes::id()` instead of inlining. This new crate costs at most 50ms of compile time on my machine

Reply:
Nice! I just remembered a discussion with Jon earlier where we agreed to have this first version depending on the published version of the crates, so I will probably disable the assert tests and keep the inline values. I will add a note about the `solana_sdk_ids` crate, so when we bump the dependencies to `2.2` we can replace the inlines.